src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -45,6 +45,16 @@
  */
 class FrameworkBundleAdminController extends Controller
 {
+    /**
+     * Used for success flash messages
+     */
+    protected const FLASH_TYPE_SUCCESS = 'success';
```

Comment:
What if you want to add a flash messages outside a Controller (like in a service / hook / module)
```suggestion
    const FLASH_TYPE_SUCCESS = 'success';
```